e2e/setup/lookout-ingester-config.yaml

```diff
@@ -15,12 +15,16 @@ pulsar:
   enabled: true
   URL: "pulsar://pulsar:6650"
   jobsetEventsTopic: "events"
+  receiveTimeout: 5s
+  backoffTime: 1s 
 
 paralellism: 1
 subscriptionName: "lookout-ingester"
 batchSize: 10000
-batchDuration: 500ms
-pulsarReceiveTimeout: 5s
-pulsarBackoffTime: 1s
+batchDuration: 500ms  
 minJobSpecCompressionSize: 1024
 userAnnotationPrefix: "armadaproject.io/"
+
+debug:
```

Comment:
So these config options are strictly necessary to get stable e2e tests for lookoutingester. They *do*, however, offer a stronger guarantee that the tests will observe every update to the jobs table.

Reply:
Do you need `forceScalarDbUpdates` because if you turn conflation off then we produce incorrect output (I could see how this could happen if you had multiple updates for the same job)?

If this is the case then I'd just have `disableConflateDBUpdates` as an option and have that internally do the forcing to scalar. Thais is because just setting `disableConflateDBUpdates`  alone would be a bad config.

Reply:
You hit the nail on the head. Yes, I can simplify this to get rid of `forceScalarDbUpdates`.